plumbing/object/commit.go

```diff
@@ -222,6 +222,72 @@ func (b *Commit) Encode(o plumbing.EncodedObject) error {
 	return err
 }
 
+// FileStat stores the status of changes in content of a file.
+type FileStat struct {
+	Name     string
+	Addition int
+	Deletion int
+}
+
+func (fs FileStat) String() string {
+	totalChanges := fs.Addition + fs.Deletion
+	adds := strings.Repeat("+", fs.Addition)
+	dels := strings.Repeat("-", fs.Deletion)
+	return fmt.Sprintf(" %s | %d %s%s", fs.Name, totalChanges, adds, dels)
```

Comment:
This output is not correct. `totalChanges` is correct, but the number of `+` and `-` is not. Should be the ratio of added and removed lines. See `man diffstat` for more info.

Also: https://stackoverflow.com/questions/7024848/git-diff-stat-explanation